packages/go/schemas/types/core/attachment.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$id": "/core/attachment",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Attachment",
+  "description": "An attachment is a file that is associated with a definition.",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "filename",
+    "content"
+  ],
+  "properties": {
+    "filename": {
+      "type": "string"
+    },
+    "contentType": {
+      "description": "The MIME type of the file.",
+      "type": "string"
+    },
+    "content": {
+      "type": "object",
+      "title": "AttachmentContent",
+      "additionalProperties": false,
+      "description": "The content of the file.",
+      "required": [
+        "type",
+        "value"
+      ],
+      "properties": {
+        "type": {
+          "title": "AttachmentContentType",
+          "type": "string",
+          "enum": [
+            "url",
+            "base64",
+            "base64-gzip",
```

Comment:
perhaps base64+gzip isn't that the style you use in http content types?

Reply:
This isn't a http content type though ?

Reply:
I know, both just for making it look familiar

Reply:
Renamed to format - you're right this was confusing